v21.1/cockroach-debug-zip.md

```diff
@@ -38,19 +19,47 @@ The file produced by `cockroach debug zip` can contain highly [sensitive, identi
+	{{site.data.alerts.end}}
 
 - If you experience severe or difficult-to-reproduce issues with your cluster, Cockroach Labs might ask you to send us your cluster's debugging information using `cockroach debug zip`.
 
-### Collecting log files
+### Files
 
-When you issue the `debug zip` command, the node that receives the request connects to each other node in the cluster. Once it's connected, the node requests the content of all log files stored on the node, the location of which is determined by the `--log-dir` value when you [started the node](cockroach-start.html).
+{{site.data.alerts.callout_success}}
+Log files, heap profiles, and goroutine dumps can greatly increase the size of the `cockroach debug zip` output. If you are collecting data from a large cluster, we recommend first experimenting with [`cockroach debug list-files`](cockroach-debug-list-files.html) and the `--exclude-files`, `--include-files`, `--files-from`, and/or `--files-until` [flags](#flags) to limit the `.zip` file size.
+{{site.data.alerts.end}}
 
-Because `debug zip` relies on CockroachDB's distributed architecture, this means that nodes not currently connected to the cluster cannot respond to the request, so their log files *are not* included. In such situations, we recommend using the [`--host` flag](#general) to point `debug zip` at individual nodes until data has been gathered for the entire cluster.
+These files collected by `cockroach debug zip` can be filtered using the `--exclude-files`, `--include-files`, `--files-from`, and/or `--files-until` [flags](#flags):
 
-After receiving the log files from all of the active nodes, the requesting node aggregates the files and writes them to an archive file you specify.
+| Information                                      | Filename                                                                             | Directory                      |
+|--------------------------------------------------|--------------------------------------------------------------------------------------|--------------------------------|
+| [Log files](configure-logs.html#log-file-naming) | `cockroach-{log-file-group}.{metadata}.log`                                          | `nodes/{node}/logs`            |
+| Goroutine dumps                                  | `goroutine_dump.{date-and-time}.{metadata}.double_since_last_dump.{metadata}.txt.gz` | `nodes/{node}/logs/goroutines` |
+| Heap profiles                                    | `memprof.{date-and-time}.{heapsize}.pprof`                                           | `nodes/{node}/heapprof`        |
+| Memory statistics                                | `memstats.{date-and-time}.{heapsize}.txt`                                            | `nodes/{node}/heapprof`        |
```

Comment:
1. you're missing one more category: CPU profiles.  It's not activated by default. When activated, the files are named `cpuprof.{date-and-time}`
2. the log files also contain a time stamp.

Reply:
@knz I want to document the CPU profiles, but should indicate how CPU profile collection is activated. what setting will cause CPU profiles to appear in `debug zip` (and `debug list-files`)?

Reply:
Actually, feel free to disregard above question, as I don't need to go into this level of detail here.